plugins/star-rating/api/api.js

```diff
@@ -2,7 +2,8 @@ var exported = {},
     common = require('../../../api/utils/common.js'),
     crypto = require('crypto'),
     countlyCommon = require('../../../api/lib/countly.common.js'),
-    plugins = require('../../pluginManager.js');
+    plugins = require('../../pluginManager.js'),
+    localize = require('../../../api/utils/localization.js');
```

Comment:
Now you don't need the localization module here then :)

Reply:
yep I removed that dependencies on my last commit.
https://github.com/Countly/countly-server/pull/880/commits/5bf4d7b5a07b01ec15fba08f4f20b7a5f7471fa5